src/Mail/SendFailedException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * @author Jan Dvořák <dvorak@spinao.cz>
+ */
+
+namespace Nette\Mail;
+
+use Nette;
+
+class SendFailedException extends Nette\InvalidStateException
```

Comment:
Better would be `SendingException` or something. You don't have to tell explicitly that it failed... it's pretty obviouse from the fact that it's an exception. Also, you problably shouldn't extend `InvalidStateException`.